integration-tests/search.test.ts

```diff
@@ -536,220 +539,148 @@ describe('searches with patient permission levels', () => {
         };
         encounterResource = (await adminClient.post('Encounter', encounterResourceJson)).data;
         await waitForResourceToBeSearchable(adminClient, encounterResource);
+
+        // create patient with linked org
+        const orgJson = {
+            resourceType: 'Organization',
+            name: 'Health Level Seven International',
+            alias: ['HL7 International'],
+        };
+        orgResource = (await adminClient.post('Organization', orgJson)).data;
+        const randPatient = randomPatient();
+        randPatient.managingOrganization.reference = `Organization/${orgResource.id}`;
+        testPatient = (await adminClient.post('Patient', randPatient)).data;
+        // wait for the patient to be asynchronously written to ES
+        await waitForResourceToBeSearchable(adminClient, testPatient);
```

Comment:
Do we need any other waits? or would we be okay

Reply:
This should be waiting for everything correctly!